src/Serialization/Upgrades/1.0.0.jl

```diff
@@ -0,0 +1,20 @@
+################################################################################
+# Upgrade Summary
+# 
+# - `ZZLatWithIsom`: we now only serialized the ambient `QuadSpaceWithIsom` and
+# a basis matrix of the underlying lattice; the upgrade recovers the basis matrix
+# in the old tree.
+
+push!(upgrade_scripts_set, UpgradeScript(
+  v"1.0.0",
+  function upgrade_1_0_0(s::UpgradeState, dict::Dict)
+
+    upgraded_dict = dict
+
+    if haskey(dict, :_type) && dict[:_type] == "ZZLatWithIsom"
+      haskey(dict[:data], :basis) || (upgraded_dict[:data][:basis] = dict[:data][:lattice][:data][:basis])
```

Comment:
```suggestion
      haskey(dict[:data], :basis) && (upgraded_dict[:data][:basis] = dict[:data][:lattice][:data][:basis])
```
I imagine you meant to apply the code on the right?

Reply:
No, because if there is already `:basis` known we do not have anything to do. In the previous serialization, we had `:ambient_space` and `:lattice` and I replaced the `:lattice` by `:basis`: so for the upgrade, I go through the old tree to recover `:basis` inside `:lattice`. But the new file already know `:basis`, so I do not have to modify anything in that case. Hope it is clear.

Reply:
ah ok, but then `=` should be  `==`?

Reply:
ah ok, i see